code/datums/uplink_item.dm

```diff
@@ -737,7 +737,7 @@ var/list/uplink_items = list()
 	name = "Syndicate Bundle"
 	desc = "Syndicate Bundles are specialised groups of items that arrive in a plain box. These items are collectively worth more than 20 telecrystals, but you do not know which specialisation you will receive."

```

Comment:
I should note I'm going through the boxes to ensure that you do get more than 30 TC's worth of items.